src/SFA.DAS.EmployerAccounts.UnitTests/Services/PayeSchemes/WhenIGetAccountPayeSchemes.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web.Util;
+using MediatR;
+using Moq;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using SFA.DAS.EmployerAccounts.Data;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Models.Levy;
+using SFA.DAS.EmployerAccounts.Models.PAYE;
+using SFA.DAS.EmployerAccounts.Queries.GetAccountPayeSchemes;
+using SFA.DAS.EmployerAccounts.Services;
+using SFA.DAS.EmployerAccounts.UnitTests.Queries;
+using SFA.DAS.HashingService;
+using SFA.DAS.Validation;
+
+namespace SFA.DAS.EmployerAccounts.UnitTests.Services.PayeSchemes
+{
+    class WhenIGetAccountPayeSchemes
+    {
+        private const long AccountId = 2;
+        private static readonly DateTime UpdateDate = DateTime.Now;
+
+        private PayeView _payeView;
+        private DasEnglishFraction _englishFraction;
+
+        private Mock<IPayeRepository> _payeSchemesRepository;
+        private Mock<IEnglishFractionRepository> _englishFractionsRepository;
+        private Mock<IHashingService> _hashingService;
+        private IPayeSchemesService SUT;
+        private string _hashedAccountId;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _payeView = new PayeView
+            {
+                AccountId = AccountId,
+                Ref = "123/ABC"
+            };
+
+            _englishFraction = new DasEnglishFraction
+            {
+                EmpRef = _payeView.Ref,
+                DateCalculated = UpdateDate,
+                Amount = 0.5m
+            };
+
+            _hashedAccountId = "123ABC";
+
+            _payeSchemesRepository = new Mock<IPayeRepository>();
+            _englishFractionsRepository = new Mock<IEnglishFractionRepository>();
+            _hashingService = new Mock<IHashingService>();
+
+            _payeSchemesRepository.Setup(x => x.GetPayeSchemesByAccountId(It.IsAny<long>())).ReturnsAsync(new List<PayeView>
+            {
+                _payeView
+            });
+
+            _englishFractionsRepository.Setup(x => x.GetCurrentFractionForSchemes(It.IsAny<long>(), It.IsAny<IEnumerable<string>>()))
```

Comment:
Are we able to remove the IsAnys and replace with specific values?

Reply:
Yes, I have just left these tests as they were have not looked at re-writing them. I am happy to look at writing better tests, just used what was already there.